arbitrator/prover/src/machine.rs

```diff
@@ -575,8 +576,9 @@ impl GlobalState {
         for item in self.bytes32_vals {
             h.update(item)
         }
-        for item in self.u64_vals {
-            h.update(item.to_be_bytes())
+        // Exclude the hotshot height
+        for i in 0..(GLOBAL_STATE_U64_NUM - 1) {
+            h.update(self.u64_vals[i].to_be_bytes())
```

Comment:
Isn't it necessary to include the height in the global state hash? Cant think of an explicit attack vector, but my intuition says that excluding this could allow some prover to backtrack during the arbitration process

Reply:
Yes actually we need to include the hotshot height in the global state. But now just for passing all the tests. It is a big project. Go, Rust, Solidity